src/common/TermModifiers.ts

```diff
@@ -0,0 +1,4 @@
+export enum TermModifiers {
```

Comment:
Do you think these are going to be used in more than one place, i.e. outside of the `modifier` prop? If not, I don't think they need their own enum, especially since they're very descriptively named to begin with